packages/core/lib/testing/testrunner.js

```diff
@@ -104,15 +104,16 @@ TestRunner.prototype.endTest = async function(mocha) {
     return;
   }
 
-  function indent(unindented, indentation, initialPrefix = "") {
-    return unindented
-      .split("\n")
-      .map((line, index) =>
-        index === 0
-          ? initialPrefix + " ".repeat(indentation - initialPrefix) + line
-          : " ".repeat(indentation) + line
+  function indent(unindented, indentation, initialPrefix = "", split = "\n") {
```

Comment:
This function is confusing now because `unindented` might now be one of two different types.  It could be a string *or* an array of strings.  If you're going to do that, I would suggest some sort of comment, and I would suggest some sort of comment explaining.

Reply:
Actually, if you make the changes I suggest below, a comment is probably unnecessary, because they'll make it clear enough what's going on.